Samples/iOS/Objective-C/IOS-SDK-Demo-App/Main.storyboard

```diff
@@ -850,7 +850,7 @@
                                             <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="89y-vo-TxN" userLabel="Native Ad View">
                                                 <rect key="frame" x="8" y="95" width="377" height="422"/>
                                                 <subviews>
-                                                    <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="Q8I-0t-sEd" customClass="MediaView">
+                                                    <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="Q8I-0t-sEd" customClass="MediaView" customModule="VungleAdsSDK">
```

Comment:
@bobVungle @crbinsf 
We might need to add this in integration doc?? Or pubs doesn't need to do this?

Reply:
If the publisher is familiar with using Storyboards, they should know how to set up the Storyboard. Additionally, I believe very few publishers would use native ads in this way.